### ISSUE TITLE
Editing old posts should send to individual post page

### ISSUE THREAD

**Issue by rtpHarry** [OpenLiveWriter/OpenLiveWriter#377]
Using 0.5.1.4 with Blogger.

When I edit an old post that was published ages ago and I press publish it sends me to the homepage of my blog.

It would make more sense to send me to the individual post url as the post in question has long passed from the blog homepage.

**Comment by lextm** [comment]
0.6.0.0 is now available. So you should first update to that release and then see if this issue remains.

**Comment by rtpHarry** [comment]
Tested in 0.6.0.0 and same behaviour occurs.

**Comment by kathweaver** [comment]
I see the behavior in Typepad.  I opened a post that I created a long time ago, and when I published it goes to the main page.

Also edited a couple of weeks ago.

Now the question is, do we want the blog to be opened at the main page or at the page of the post.

Both of my posts opened to the main page of the blog;

**Comment by rtpHarry** [comment]
There is already an option in there to toggle the behaviour:

![image](https://cloud.githubusercontent.com/assets/1038062/13164940/d08ccdf2-d6b0-11e5-9a0d-adc788b5b7d0.png)

If it's decided that not everyone would want to go to the single page post then it could be split into a checkbox + sub radio buttons, or even a triple radio button list like the element above it.

**Comment by kathweaver** [comment]
Yes, I have that checked and it is taking me to the front page, not the post I updated.

**Comment by lextm** [comment]
I don't have time this weekend to dig further into the issue, but I came across it also myself so it is annoying. The strange behavior can be explained by the source code, where if a permanent link cannot be detected during publishing, then Open Live Writer opens the home page instead of individual blog post page.

https://github.com/OpenLiveWriter/OpenLiveWriter/blob/master/src/managed/OpenLiveWriter.PostEditor/BlogPostEditingManager.cs#L648

Just need to track down the process on how the permanent link should be queried during publishing, then this issue should be resolved.

Anyone who is interested in contributing should be able to create a pull request once fixing it. I will attempt to fix it if nobody does it by the end of March.

**Comment by ScottIsAFool** [comment]
Interesting, the wording of that setting is incorrect (it seems), the wording should be that it should open the blog, not the post.

**Comment by kathweaver** [comment]
I have a pull request to change the wording to what actually happens: #380

**Comment by lextm** [comment]
@ScottIsAFool I don't think it is a wording issue. 

I provided an easier fix in #382, where if a permanent link exists Open Live Writer opens it instead of the homepage.

I don't know why the original code checks `IsPage`, but that check does not seem to be necessary to me, and it prevents the behavior from matching the expectation (from many users).

**Comment by ScottIsAFool** [comment]
I'm not saying the behaviour you want isn't valid, just that if you look at the wording on the dialog when a post/page is being published, the check box there says about opening the blog. That setting corresponds to the one in the settings shown in the screenshot above. I think this should be a separate option. The expectation of the user is only there because of the incorrect wording ;)

**Comment by SalvorinFex** [comment]
Question;  How do you open previous posts on a Blogger account?

**Comment by ScottIsAFool** [comment]
@SalvorinFex press ctrl + O to show the open post window, then choose your blogger blog from the list on the left. You can access your older blog posts from there.

**Comment by SalvorinFex** [comment]
That worked great, thank you!

**Comment by SalvorinFex** [comment]
That worked Great!

Thank you!

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
